lib/cli.ex

```diff
@@ -1,10 +1,44 @@
 defmodule TicTacToe.CLI do
+  alias TicTacToe.Board
+  alias TicTacToe.CLI
 
   def welcome() do
     IO.puts "Welcome to Tic Tac Toe!"
   end
 
+  def print(message) do
+    IO.puts message
+  end
+
+  def get_display_list(spaces) do
+    for x <- [1, 2, 3, 4, 5, 6, 7, 8, 9] do
+      if Board.space_is_available(spaces, x) do
+        x
+      else
+        Enum.at(spaces, x - 1)
+      end
+    end
+  end
+
```

Comment:
This function knows that board spaces is an array -- should it belong here? 
Generally, data structures used within a module and any functions that relies on that knowledge should be kept together within the same module to help with decoupling.

Reply:
Makes sense! [I moved it to the board module](https://github.com/digitopolis/tic-tac-toe-elixir/pull/4/commits/ad347cd5b689acf8615438fb1c309843aa4426d0)